frontend/src/metabase/parameters/utils/dashboards.ts

```diff
@@ -133,15 +144,21 @@ function buildFieldFilterUiParameter(
   parameter: Parameter,
   mappings: ExtendedMapping[],
   metadata: Metadata,
+  questions: Record<CardId, Question>,
 ): FieldFilterUiParameter {
   const mappingsForParameter = mappings.filter(
     mapping => mapping.parameter_id === parameter.id,
   );
-  const mappedFields = mappingsForParameter.map(mapping => {
+  const uniqueMappingsForParameters = _.uniq(mappingsForParameter, mapping =>
+    mapping.target.toString(),
+  );
```

Comment:
I'm worried this can be dangerous and unpredictable. For example, a mapping target like that:

```js
const field_ref = [ "field", 1, { 'base-type': 'type/Integer', 'source-field': 40 } ]
const target = [ "dimension", field_ref ]
```

will turn into:

```js
"dimension,field,1,[object Object]"
```

Even though we should most likely ignore this options object when comparing field refs, I'm worried this could be too implicit

IMO we could use something like `_.isEqual` to find unique mappings (probably not gonna work with `_.uniq` tho). Or ideally have a CLJS method to do that

Reply:
Fixed a91692bf68a11923b3f35d8615c9af29bdbe09ee

Based on my testing, we care only about the first two elements of the target, but let's stay on the safe side and improve it later